signingvalidation.proj

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <Target Name="Build"
+          DependsOnTargets="BuildPrerequisites;
+                            Validate" />
+
+  <Target Name="BuildPrerequisites"
+          DependsOnTargets="CreateVersionInfoFile;
+                            BuildCustomTasks" />
+
+  <Target Name="Validate">
+    <MSBuild Projects="$(MSBuildThisFileDirectory)tools-local\signingvalidation\dir.proj" />
```

Comment:
Is there value in making this a separate project vs just making it a .targets file that gets imported?

Also, I really don't like this pattern of naming files `dir.proj`.  It makes it too confusing.  There's a bunch of "dir.proj" files in the repo, and when I have a few of them open in a text editor, it is impossible to tell which one is which.  It also makes it impossible to reference the file without giving the full path - which dir.proj file are you referring to?